### ISSUE TITLE
Error in htmlwidgets::createWidget unused argument

### ISSUE THREAD

**Issue by scizmeli** [bokeh/rbokeh#70]
Hello I just discovered Rbokeh. It looks very nice indeed. 

To make some tests, I freshly installed rBokeh and htmlwidgets for my R 3.2.0 under Linux. When I try the example in the homepage, I get the following error : 

    figure() %>% ly_points(1:10) %>%
    tool_lasso_select()
    Error in htmlwidgets::createWidget(name = "rbokeh", x = list(spec = spec,  : 
    unused argument (preRenderHook = rbokeh_prerender)

Is there anything I'm doing wrong? Can you please help?
thanks

**Comment by hafen** [comment]
Right now you need to install htmlwidgets from github, not CRAN.  This will change soon when the new version of htmlwidgets is pushed to CRAN.

```
devtools::install_github("ramnathv/htmlwidgets")
```

**Comment by scizmeli** [comment]
This is exactly how I had installed htmlwidgets,  in the way it was described in homepage. Just to be sure, I reinstalled htmlwidgets... But still the same issue...

I also figured out that the I get the error when I run the command on rstudio (latest version:  0.98.1103) but the same command works well when I run R from the linux command line.
So there is a glitch in the rstudio environment...

Any more input I can provide so we can identify and  solve the issue?
